### ISSUE TITLE
Request accessibility/508 review for Outreach and Events on VA.Gov

### ISSUE THREAD

**Issue by jefflbrauer** [department-of-veterans-affairs/va.gov-vfs-teams#31]
## Helpful Links

- [508 Accessibility Issue
  Template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=508-issue.md&title=)

**Comment by 1Copenut** [comment]
@jefflbrauer When you've got a few minutes, would you mind updating this issue ticket with some information for each section? I use the staging URL, creds, test plan, etc. to scope and size audits, and plan my sprints. Please let me know if you'd like to review this further.

/cc @andreahewitt-odd

**Comment by jefflbrauer** [comment]
Odd- I had entered this data in the ticket. 

URL: staging.va.gov/outreach-and-events
No credentials required as this is public facing. 

The scope is the above landing page and two sub pages within (3 total) 

Thank you,
Jeff

**Comment by 1Copenut** [comment]
@jefflbrauer Ah! I went through the edit screen and your text was there. The `<!-- text -->` wrapper was causing them to be rendered as comments, so they didn't show up. That's on me, and a chance to improve my template instructions.

**Comment by jefflbrauer** [comment]
Hi @1Copenut Did this scan ever get completed?

**Comment by 1Copenut** [comment]
Hi @jefflbrauer . We're under way and should be done auditing on Monday or Tuesday.

**Comment by 1Copenut** [comment]
@jefflbrauer I just assigned an issue to you for this work. I'll add you to any additional issues we find as the primary assignee.
